extensions/positron-assistant/src/md/prompts/chat/mapedit.md

```diff
@@ -3,4 +3,5 @@ You will be given a document and a block. Output a JSON array containing objects
 For example, for the input { "document": "a\nb\nc\nd\ne", "block": "b\n123\ne" } output [{ "delete": "b\nc\nd\ne", "replace": "b\n123\ne" }].
 
 If it is not clear where the block should go, output an { "append": string } object in the array to add it to the end of the document.
-Return ONLY the JSON string, nothing else.
+Return ONLY the JSON string, nothing else. Do NOT use a code fence, return the JSON as plain output.
```

Comment:
I've found that putting examples in tags can help with this sort of problem – maybe it makes it clearer that no code fence is needed. I originally saw that in the [Anthropic system prompts](https://github.com/asgeirtj/system_prompts_leaks/tree/main/Anthropic). For instance we could remove the `For example, ...` line above and put this somewhere:

```
<example>
<user>{ "document": "a\nb\nc\nd\ne", "block": "b\n123\ne" }</user>
<response>[{ "delete": "b\nc\nd\ne", "replace": "b\n123\ne" }]</response>
<example>
```